dev/PersonPicture/PersonPicture.xaml

```diff
@@ -19,9 +19,26 @@
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
                                 <!-- Visual State when a Photo is available for display -->
-                                <VisualState x:Name="Photo" />
-                                <!-- Visual State when no Photo is available (even if no initials are available).-->
+                                <VisualState x:Name="Photo">
+                                    <VisualState.Setters>
+                                        <Setter Target="PersonPictureEllipse.Fill" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.EffectiveImageBrush}"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <!-- Visual State when no Photo is available (but there are initials).-->
                                 <VisualState x:Name="Initials" />
+                                <VisualState x:Name="NoPhotoOrInitials">
+                                    <VisualState.Setters>
+                                        <Setter Target="InitialsTextBlock.FontFamily" Value="Segoe MDL2 Assets"/>
+                                        <Setter Target="InitialsTextBlock.Text" Value="&#xE77B;"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <!-- Visual State when in Group mode -->
+                                <VisualState x:Name="Group">
+                                    <VisualState.Setters>
+                                        <Setter Target="InitialsTextBlock.FontFamily" Value="Segoe MDL2 Assets"/>
```

Comment:
>Segoe MDL2 Assets [](start = 93, length = 17)

Why do we need "Segoe MDL2 Assets" other than "TemplateBinding FontFamily" here?

Reply:
Because the default for the control is Segoe UI -- in the case of Group display or no Initials then we need to display a symbol. So in those VSM states we temporarily override the FontFamily and Text display. Otherwise we just use the value inherited from Control.